pure_pursuit:
  ros__parameters:
    wheelbase: 1.3
    lookahead_distance: 4.5 
    control_frequency: 30
    
    # 동적 Lookahead Distance 파라미터
    # ld_gain: 0.5                  # Ld = k * speed 의 비례 상수 k
    # min_lookahead_distance: 2.5   # 최소 Ld 값
    # max_lookahead_distance: 10.0  # 최대 Ld 값
